src/util/dateUtils.ts

```diff
@@ -17,6 +17,14 @@ export const isValidDate = (date: string | number | Date | dayjs.Dayjs | null |
   return dayjs(date).isValid();
 };
 
+export function onTulevaisuudessa(date: string | null | undefined): boolean {
+  if (!date) {
+    return false;
+  }
+  let parsedDate = dayjs(date);
+  return parsedDate.isBefore(dayjs());
```

Comment:
Tässä tutkitaan väärin päin päivämääriä. Lisäksi minusta ei ole tarpeen jättää tätä frontille, vaan kerran vaivaudutaan bäkkärissä kaivamaan myöhäisin vuorovaikutusaika, niin samalla vaivalla olisi voinut palauttaa onko se tulevaisuudessa vai ei.

Ja sitten jos se tänne jätetään niin a) tarvitaanko erillistä utility-funktiota, kun nuo dayjs:n funkkarit ovat jo olemassa b) jos tehdään oma, niin pitäisikö sillä olla sitten testit vai ei